pkg/aws/client.go

```diff
@@ -14,8 +14,8 @@ import (
 
 const (
 	kAWSZoneID = "AWS_ZONE_ID"
-
 	AliasTarget = "AliasHostedZoneID"
+	defaultTTL = 60 // Default TTL for DNS records in seconds if not specified
```

Comment:
Like you said in the comment below, I think this should be an `int64` as it makes it code a bit more straight forward and type conversion only needs to happen when replacing the default values.

Reply:
done :)